index.html

```diff
@@ -6,12 +6,12 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="./css/main.css" />
     <link rel="icon" href="./favicon.ico" />
-    <title>Aziz Mejri - Portfolio</title>
+    <title>David Tamayo - Portfolio</title>
   </head>
   <body>
     <header>
-      <h1 id="logo">Aziz Mejri</h1>
-      <button type="button" class="menu-bars">
+      <h1 id="logo">David Tamayo</h1>
+      <button type="button" id="menu-bars" class="menu-bars">
         <i class="fas fa-bars"></i>
       </button>
```

Comment:
- [ ] On desktop size the hamburger menu button is still visible. This feature is only meant for the mobile view so please make changes to it so it disappears when in desktop view. :100: